src/resources/views/signup.ejs

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+  <title>DSC-DUT | Shorten URL tool</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Google+Sans&display=swap" rel="stylesheet" />
+  <link rel="stylesheet" href="/css/style.css" />
+  <link rel="stylesheet" href="/css/loader.min.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+    integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="shortcut icon" href="/img/favicon.ico" type="image/x-icon" />
+</head>
+
+<body>
+  <div id="app">
+    <div id="app-logo">
+      <img src="/img/logo.png" alt="DSC-DUT Logo" />
+    </div>
+    <a id="app-button-shorten" class="app-button shorten" type="submit" href="./">Shorten URL</a>
+    <div id="app-container">
+      <h1>Sign up to manage your URLs</h1>
+      <form id="app-form">
+        <input type="email" class="app-input" id="app-input-email" placeholder="Input your email" />
+        <div class="app-input-container">
+          <input type="password" class="app-input password" id="app-input-password-1" placeholder="Input your password" />
+          <i id="eye-password-1" onClick="togglePassword1()" class="eye-password eye-signup-1 far fa-eye-slash"></i>
+        </div>
+        <div class="app-input-container">
+          <input type="password" class="app-input password" id="app-input-password-2" placeholder="Input your password again" />
+          <i id="eye-password-2" onClick="togglePassword2()" class="eye-password eye-signup-2 far fa-eye-slash"></i>
+        </div>
+        <div id="error-password" class="error-password">
+          <i class="fas fa-exclamation"></i>
+          <span>Passwords do not match.</span>
+        </div>
+        <button id="app-button-submit" class="app-button sign" type="submit">
+          Sign up
+        </button>
+        <div class="app-link-sign">
+          <span>Have already an account? </span>
+          <a href="./signin" class="link-sign">Sign in</a>
+        </div>
+      </form>
+    </div>
+    <div id="app-loader" class="animate__animated animate__fadeOut">
+      <div class="loader">
+        <div class="dot dot1"></div>
+        <div class="dot dot2"></div>
+        <div class="dot dot3"></div>
+        <div class="dot dot4"></div>
+        <div class="dot dot5"></div>
+        <div class="dot dot6"></div>
+        <div class="dot dot7"></div>
+      </div>
+    </div>
```

Comment:
Consider use EJS partial to reuse this piece of code
See blog: https://www.digitalocean.com/community/tutorials/how-to-use-ejs-to-template-your-node-application